### ISSUE TITLE
Is there any way to toggle Accept Call Queue Calls through the code?

### ISSUE THREAD

**Issue by duyendoan** [ringcentral/ringcentral-embeddable#602]
I turn off the Accept Call Queue Call toggle before updating the widget status from Available to DND . And I'm able to get the dndStatus "DoNotAcceptDepartmentCalls". When I update the status back to Available using postMessage/clicking on the status, the toggle ACQC is also turned on automatically. Is there a way to keep it off? Or in other words, is it possible to use postMessage to update the dndStatus for this toggle?

**Comment by embbnux** [comment]
Hi @duyendoan, good catch. Currently the widget doesn't have API to set dndStatus into `DoNotAcceptDepartmentCalls`.
Let me add it into backlog.
For other status, you can use following code:
```js
document.querySelector("#rc-widget-adapter-frame").contentWindow.postMessage({
   presenceType: "DoNotAcceptAnyCalls",
   type: "rc-adapter-presenceItemClicked"
}, "*")
```

**Comment by embbnux** [comment]
Hi @duyendoan we have an update at master build to support setting dnd status. Can you help to try it?
You can follow the docs https://github.com/ringcentral/ringcentral-embeddable/blob/master/docs/control-widget.md#set-presence in master build https://ringcentral.github.io/ringcentral-embeddable/

**Comment by embbnux** [comment]
released with 1.8.2
